openmetadata-ui/src/main/resources/ui/src/components/Explore/ColumnSummary/ColumnSummary.component.tsx

```diff
@@ -38,7 +39,7 @@ export default function ColumnSummary({ columns }: ColumnSummaryProps) {
   }, [columns]);
 
   return (
-    <Space direction="vertical">
+    <Space className=" w-full" direction="vertical">
```

Comment:
I would recommend to use Row/Col for vertical aligned content

Reply:
Replaced such Space with Rox/Col